assets/src/block-editor/helpers/index.js

```diff
@@ -372,31 +372,52 @@ setUpInspectorControls.propTypes = {
 /**
  * Get AMP Layout select control.
  *
+ * @deprecated As of v2.1. Blocks with the `ampLayout` attribute will still be able to use the control.
+ *
  * @param {Object} props Props.
  *
  * @return {ReactElement} Element.
  */
-const AmpLayoutControl = ( props ) => {
+export const AmpLayoutControl = ( props ) => {
 	const { name, attributes: { ampLayout }, setAttributes } = props;
 
+	if ( undefined === ampLayout ) {
```

Comment:
Per https://wordpress.org/support/topic/seeing-a-the-amp-layout-setting-is-deprecated-on-images/ this may need to be changed to:

```js
if ( undefined === ampLayout || '' === ampLayout ) {
```